website/static/js/pages/project-base-page.js

```diff
@@ -32,4 +32,40 @@ $(document).ready(function() {
     $.getJSON(node.urls.api, function(data) {    
         $('body').trigger('nodeLoad', data);
     });
+
+    var target = 50, smallScreenSize = 767,
+    timeout = null;
+    var checkPanelPosition = function() {
+        var bodyWidth = $(document.body).width();
+        var scrollTopPosition = $(window).scrollTop();
+        if (bodyWidth <= smallScreenSize) {
+            if (scrollTopPosition >= target) {
+                $('.cp-handle').css('margin-top', 50);
+            }
+            else {
+                $('.cp-handle').css('margin-top', 95 - scrollTopPosition);
+            }
+        }
+    };
+
+    checkPanelPosition(); /* Init when refreshing the page*/
+    $(window).scroll(function () {
+        if (!timeout) {
+            timeout = setTimeout(function () {
+                clearTimeout(timeout);
```

Comment:
@samanehsan  Firstly, the reason why call clearTimeOut inside of setTimeout is to avoid running setTimeout when user doesn't scrolling. Only when scrolling, there will be setTimeout. As for  `timeout = null`, basically, variable `timeout` is id of settimeout. What clearTimeout() does is to get the id and stop the timeout function. But it doesn't set  timeout variable as `null`. We need timeout to be `null` so that another scroll event will create a new one.